app-arch/unarr/unarr-1.0.0.recipe

```diff
@@ -8,58 +8,84 @@ HOMEPAGE="https://github.com/sumatrapdfreader/sumatrapdf/tree/master/ext/unarr"
 COPYRIGHT="The Unarchiver project (https://code.google.com/p/theunarchiver/) \
 Simon Bünzli (zeniko at gmail.com, http://www.zeniko.ch/#SumatraPDF)"
 LICENSE="GNU LGPL v3"
-REVISION="2"
-SOURCE_URI="http://download.opensuse.org/repositories/home:/selmf/xUbuntu_16.04/libunarr_1.0~r188.c4136ab.orig.tar.xz"
-CHECKSUM_SHA256="d799a8f61e2c87c5fb52f01770b11eaaff1e3cb5840ab6852056d15967f664cb"
-SOURCE_DIR="libunarr"
+REVISION="3"
+SOURCE_URI="https://github.com/selmf/unarr/archive/v${portVersion}.tar.gz"
+CHECKSUM_SHA256="ef1610d636641751ca234da7998b3f6b53313c7e526492a6070dd572b99d4685"
 
 ARCHITECTURES="!x86_gcc2 x86 x86_64"
 SECONDARY_ARCHITECTURES="x86"
 
 PROVIDES="
-	unarr$secondaryArchSuffix = $portVersion
-	cmd:unarr$secondaryArchSuffix = $portVersion
 	lib:libunarr$secondaryArchSuffix = $portVersion
 	"
+
 REQUIRES="
 	haiku$secondaryArchSuffix
+	lib:libbz2$secondaryArchSuffix
+	lib:liblzma$secondaryArchSuffix
+	lib:libz$secondaryArchSuffix
 	"
 
 PROVIDES_devel="
 	unarr${secondaryArchSuffix}_devel = $portVersion
 	devel:libunarr$secondaryArchSuffix = $portVersion
 	"
+
 REQUIRES_devel="
-	unarr$secondaryArchSuffix == $portVersion base
```

Comment:
This is also required.

Reply:
oops, didin't see these lines where lost.

btw, how to only produce a _devel package? Cause now the main package have not content at all, while the _devel one have the static library, headers and pkgconfig file... ?
Maybe I could build the shared library too, though.

Reply:
Usually only the main package contains everything, no devel package. Package name unarr, provides devel:linunarr.

Reply:
Fix broken recipe, build the library both in static and shared versions and move everything into main package, no more devel package.